src/games/strategy/triplea/ai/proAI/ProCombatMoveAI.java

```diff
@@ -1630,4 +1686,17 @@ private void logAttackMoves(final List<ProTerritory> prioritizedTerritories) {
       }
     }
   }
+
+  private boolean canAirSafelyLandAfterAttack(final Unit unit, final Territory t) {
+    final boolean isAdjacentToAlliedFactory =
+        Matches.territoryHasNeighborMatching(data,
+            ProMatches.territoryHasInfraFactoryAndIsAlliedLand(player, data)).match(t);
+    final int range = TripleAUnit.get(unit).getMovementLeft();
+    final int distance =
+        data.getMap().getDistance_IgnoreEndForCondition(ProData.unitTerritoryMap.get(unit), t,
```

Comment:
Would you mind adding better APIs, and calling those?
- `getDistance_IgnoreEndForCondition` is not lower camel cased
- `territoryCanMoveAirUnitsAndNoAA` takes a boolean parameter, a new API with no `boolean` parameter can replace the call and will simplify the client. Alternatively please replace the `boolean` parameter with an `Enum`